arci/src/lib.rs

```diff
@@ -10,7 +10,6 @@ mod waits;
 pub use async_trait::async_trait;
 pub use clients::*;
 pub use error::*;
-// re-export
 pub use nalgebra::{Isometry2, Isometry3};
 pub use traits::*;
 pub use waits::*;
```

Comment:
re-exporting only `async_trait` and `nalgebra`.
Others are inside of this crate.

How about sorting?

```rust
pub use clients::*;
pub use error::*;
pub use traits::*;
pub use waits::*;

// re-export
pub use async_trait::async_trait;
pub use nalgebra::{Isometry2, Isometry3};
```

Reply:
FYI:
> How about sorting?
The order is forced by the rustfmt setting.

Reply:
Oh, I understand that it's not possible to sort!
(I wanted to add comments about these lines are exporting **external crates**)

Reply:
> I wanted to add comments about these lines are exporting external crates

Oh, I didn't notice it was intentional.

> Currently the order is forced by the rustfmt setting.

FWIW, the workaround for this rustfmt issue is to add the `crate::` or `self::` prefix. It will be formatted as follows:

```rust
// re-export
pub use async_trait::async_trait;
pub use nalgebra::{Isometry2, Isometry3};

pub use crate::{clients::*, error::*, traits::*, waits::*};
```

refs: https://github.com/rust-lang/rustfmt/issues/4709

Reply:
I think we can use your proposal (use crate::).

Reply:
Ok, updated to group exports from modules.